shared/unattended/SLES-12.xml

```diff
@@ -461,7 +461,13 @@
   <partitioning config:type="list">
     <drive>
       <initialize config:type="boolean">true</initialize>
-      <partitions config:type="list"/>
+      <partitions config:type="list">
+        <partition>
+          <mount>/</mount>
+          <filesystem  config:type="symbol">xfs</filesystem>
```

Comment:
I remember that SLES 12 would use btrfs as the default filesystem type, so is there some reason to replace it?

Reply:
yes, it's in the commit message. The zipl which is used on s390 as a bootloader can't boot from btrfs (which took me quite a long time to find out). Later I found presentation from 2015 where they say btrfs nor xfs are supported, but apparently xfs is now).

Reply:
Sorry, I just missed that.